test/jdk/java/awt/Focus/FocusTraversalPolicy/ButtonGroupLayoutTraversal/ButtonGroupLayoutTraversalTest.java

```diff
@@ -58,71 +58,77 @@
 
     public static void main(String[] args) throws Exception {
 
-        SwingUtilities.invokeAndWait(() -> changeLAF());
-        SwingUtilities.invokeAndWait(() -> initLayout(nx, ny));
-        Robot robot = new Robot();
-        robot.setAutoDelay(100);
-        robot.waitForIdle();
-        robot.delay(200);
-
-        for (int i = 0; i < nx * ny - nx * ny / 2 - 1; i++) {
-            robot.keyPress(KeyEvent.VK_RIGHT);
-            robot.keyRelease(KeyEvent.VK_RIGHT);
-        }
+        try {
+            SwingUtilities.invokeAndWait(() -> changeLAF());
+            SwingUtilities.invokeAndWait(() -> initLayout(nx, ny));
+            Robot robot = new Robot();
+            robot.setAutoDelay(100);
+            robot.waitForIdle();
+            robot.delay(1000);
+
+            for (int i = 0; i < nx * ny - nx * ny / 2 - 1; i++) {
+                robot.keyPress(KeyEvent.VK_RIGHT);
+                robot.keyRelease(KeyEvent.VK_RIGHT);
+                robot.waitForIdle();
+            }
 
-        for (int i = 0; i < nx * ny / 2; i++) {
-            robot.keyPress(KeyEvent.VK_TAB);
-            robot.keyRelease(KeyEvent.VK_TAB);
-        }
+            for (int i = 0; i < nx * ny / 2; i++) {
+                robot.keyPress(KeyEvent.VK_TAB);
+                robot.keyRelease(KeyEvent.VK_TAB);
+                robot.waitForIdle();
+            }
 
-        robot.waitForIdle();
-        robot.delay(200);
+            robot.waitForIdle();
```

Comment:
Looks like this is not needed, a line above we call it already(in the loop.)

Reply:
Removed..